CHANGELOG.md

```diff
@@ -353,6 +353,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (abci) [#15845](https://github.com/cosmos/cosmos-sdk/pull/15845) Remove duplicating events in `logs`
 * (baseapp) [#15519](https://github.com/cosmos/cosmos-sdk/pull/15519/files) BeginBlock & EndBlock events have begin or endblock in the events in order to identify which stage they are emitted from since they are returned to comet as FinalizeBlock events, 
 * (store/streaming) [#15519](https://github.com/cosmos/cosmos-sdk/pull/15519/files) State Streaming removed emitting of beginblock, endblock and delivertx in favour of emitting FinalizeBlock. 
+* (x/bank) [#17273](https://github.com/cosmos/cosmos-sdk/pull/17273) Remove message events including `sender` attribute whose information is already present in the relevant events
```

Comment:
```suggestion
* (x/bank) [#17273](https://github.com/cosmos/cosmos-sdk/pull/17273) Remove `EventTypeMessages` from BaseSendKeeper's `InputOutputCoins` and `SendCoins` method.
```
This log doesn't specify where does it apply. Something like this would be more clear.

Reply:
It does specify the location of the change, but does not specify the reason of this refactor.

What should I prioritize? Or should I include all of the information? @kocubinski @julienrbrt

Reply:
Both works for me, but that's good that there is the reason.